src/cascadia/TerminalSettingsEditor/CommonResources.xaml

```diff
@@ -159,4 +159,58 @@
         <Setter Property="Width" Value="{StaticResource StandardBoxMinWidth}" />
         <Setter Property="HorizontalAlignment" Value="Left" />
     </Style>
+    <!-- DELETE BUTTON STYLING IN COLORSCHEMES.XAML FILE -->

```

Comment:
What's this comment about? Shouldn't we be removing this styling from the other location if it's going to be here as common?